openmdao/utils/om.py

```diff
@@ -280,6 +281,40 @@ def _tree_setup_parser(parser):
                         help="Show which components compute approximations.")
 
 
+def _clean_setup_parser(parser):
+    """
+    Set up the openmdao subparser for the 'openmdao clean' command.
+
+    Parameters
+    ----------
+    parser : argparse subparser
+        The parser we're adding options to.
+    """
+    parser.add_argument('path', nargs='*', default='.',
+                        help='Path(s) from which OpenMDAO output directories should be remove.')
+    parser.add_argument('-f', '--force', action='store_true', dest='force',
+                        help='Remove output directories without confirmation.')
+    parser.add_argument('-n', '--norecurse', action='store_true', dest='no_recurse',
+                        help='Do not recurse into subdirectories to find directories to remove.')
+    parser.add_argument('-d', '--dryrun', action='store_true', dest='dryrun',
+                        help='Highlight directories to be removed but do not actually remove them.')
```

Comment:
I think maybe prompt should be in the parser args.  It probably should also default to True.

I am a little worried about accidentally running `openmdao clean` at the root directory and accidentally deleting years worth of runs without realizing it.

Reply:
Right now `prompt` is on by default, and the `--force` argument is the same as `do not prompt`.  I'll change it to `--noprompt` for consistency.